mcs/class/corlib/corlib_xtest.dll.sources

```diff
@@ -158,8 +158,6 @@
 ../../../external/corefx/src/System.Runtime/tests/System/String.SplitTests.cs
 ../../../external/corefx/src/System.Runtime/tests/System/NullableTests.cs
 ../../../external/corefx/src/System.Runtime/tests/System/Reflection/BindingFlagsDoNotWrap.netcoreapp.cs
-../../../external/corefx/src/System.Runtime/tests/System/Reflection/MemberInfoTests.cs
-../../../external/corefx/src/System.Runtime/tests/System/Reflection/MemberInfoTests.netcoreapp.cs
```

Comment:
Is this correct?

Reply:
@marek-safar when I updated corefx I also got some Maxim's changes so those tests don't compile (I guess Maxim has fixed it in his PR so these changes will no longer needed after I rebase my PR).